lib/resources/r4/related_person.yaml

```diff
@@ -61,9 +61,19 @@ fields:
     - display: Millennium Person Relationship Types
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/351
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-351-person-relationship-types
-    - display: Millennium Person Relationships
-      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/40
-      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-40-person-relationships
+
+- name: relationship relation extension
+  type: CodeableConcept
+  description: The nature of the relationship between a patient and the related person.
+  url: https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.relationship
+  terminology_name: relationship.extension[x].valueCodeableConcept
+  action: terminology
+  binding:
+    description: The nature of the relationship between a patient and the related person. This binding is used in <code>relationship-relation</code> extensions.
```

Comment:
relationship-relation is not the name of the extension. I guess it should be 'relation'

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/624/commits/7532319ad69a4ed7dc8e889132d3547cbd62f59c#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R72